native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -423,7 +427,23 @@ void EventDispatcher::removeAllEventListeners() {
     hashListenerId = 1;
 }
 
-void EventDispatcher::dispatchCustomEvent(const CustomEvent &event) {
+void EventDispatcher::dispatchCustomEvent(const char *eventName, int argNum, ...) {
+    if (!eventName || !strcmp(eventName, "")) {
```

Comment:
May be it is better to use assert. And is it need to check `""`?

Reply:
it's better to just return if eventName is nullptr or empty, it's not an error but a code with forgivable usage.

Reply:
As this function will be invoked many times a frame. So i suggest to use asset instead. I think it is an error.

Reply:
replaced with CCASSERT

Reply:
please use CC_ASSERT instead, CCASSERT is removed in v3.6.